src/components/organisms/NewResourceWizard/NewResourceWizard.tsx

```diff
@@ -91,6 +129,41 @@ const NewResourceWizard = () => {
     [registeredKindHandlers, resourceMap]
   );
 
+  const generateExportFileName = async () => {
+    if (fileMap[ROOT_FILE_ENTRY] && selectedFolder.startsWith(fileMap[ROOT_FILE_ENTRY].filePath)) {
+      const currentFolder = selectedFolder.split(`${fileMap[ROOT_FILE_ENTRY].filePath}`).pop();
+
+      if (currentFolder) {
+        setSelectedFolder(currentFolder.slice(1));
+      } else {
+        setSelectedFolder(ROOT_FILE_ENTRY);
+      }
+      return;
+    }
+
+    let selectedFolderResources;
+    if (selectedFolder === ROOT_FILE_ENTRY) {
+      selectedFolderResources = Object.values(resourceMap).filter(
+        resource => resource.filePath.split('/').length === 2
+      );
+    } else {
+      selectedFolderResources = Object.values(resourceMap).filter(
+        resource => resource.filePath.split('/').length > 2 && resource.filePath.startsWith(`/${selectedFolder}/`)
```

Comment:
Same here with `path.sep` instead of `/`. Also we should not check if the resource filePath starts with the selected folder as it will filter also all resources from all the subfolder from the selected folder, which is not the desired behaviour.

We could do something like this:

    Object.values(resourceMap).filter(resource => getDirname(resource.filePath).endsWith(selectedFolder));

Which will filter all resources whose parent folder ends with the selected folder.

Reply:
Incorporated.